hw3/01-characters.js

```diff
@@ -0,0 +1,38 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector("section");
+
+document.onload = getApiData(url);
+
+function getApiData(url) {
+  fetch(url).then(response => response.json().then(data => {
+    data.forEach(character => {
+      section.appendChild(makeCharacterCard(character.fullName, character.title, character.imageUrl));
+    });
+  }));
+}
+
+function makeCharacterCard(characterName, characterTitle, imageUrl) {
+  let newCard = document.createElement("div");
+  let image = document.createElement("img");
+  let name = document.createElement("p");
+  let title = document.createElement("p");
+
+  image.src = imageUrl;
+  name.textContent = characterName;
+  title.textContent = characterTitle;
+
+  newCard.classList.add("character");
+  newCard.classList.add("text-center");
+  newCard.classList.add("ms-2");
+  name.classList.add("fw-bolder");
+  name.classList.add("fs-4");
+  title.classList.add("fw-bold");
+  image.classList.add("charImg");
+
+  newCard.appendChild(image);
+  newCard.appendChild(name);
+  newCard.appendChild(title);
```

Comment:
Using the regular append() method all three child elements (image, name, and title) could be appended to newCard at the same time. This is not necessary, but it would save a few lines while still being readable.

The append() method works like appendChild(), but lets you pass in multiple items.

```suggestion
  newCard.append(image, name, title);
```